org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLCompletions.java

```diff
@@ -370,8 +370,7 @@ private void createPrologCompletionSuggestion(int tagNameEnd, String tag, Comple
 				Range editRange = getReplaceRange(tagNameEnd, end, request);
 				item.setTextEdit(new TextEdit(editRange, " version=\"1.0\" encoding=\"UTF-8\"?" + closeTag + "$0"));
 			} catch (BadLocationException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
+				LOGGER.log(Level.SEVERE, "While performing getReplaceRange for prolog completion.", e);

```

Comment:
Remove this code which is not part in this issue (I have already fixed and it will resolve conflict).